src/cityreader/cityreader.py

```diff
@@ -17,17 +22,22 @@
 cities = []
 
 def cityreader(cities=[]):
+ with open('cities.csv', newline='') as csvfile:
+  citylist = csv.reader(csvfile)
+  for row in citylist:
+   i = 1
+   cities.append(City(row[i], row[i], row[i]))
```

Comment:
It could help to break down each column here into variables. I'd also check your output against the test file output and pay special attention to the data types of each column. Everything else looks really good 👍